### PR TITLE
xGov-71: Milestone 2 - ASA Stats tax reporting reference

### PR DESCRIPTION
**Pull request by ipaleka**
The ASA Stats team will develop and deploy endpoints that will be used for creating CSV files ready to be imported into crypto tax software.

### CODE REVIEW

**Comment by ipaleka** [comment]
@SudoWeezy This proposal file name is by mistake merged under the file name ```xgov-x.md` and it has to be renamed ASAP to `xgov-71.md`.

Thank you!

https://github.com/algorandfoundation/xGov/tree/main/Proposals

**Comment by ipaleka** [comment]
We have developed and deployed a completely new historic price engine for the purpose and that endeavor has prolonged our planned deployment.

The alpha version is available on our website https://www.asastats.com . Please use the input box to enter either a single address or a bundle consisting of up to five addresses (click the expand button on the left of the input box to allow entering more than one address). Upon submitting, you'll be redirected to the address or bundle page:

https://www.asastats.com/2EVGZ4BGOSL3J64UYDE2BUGTNTBZZZLI54VUQQNZZLYCDODLY33UGXNSIU
https://www.asastats.com/540A5D8CEC896E073F9170AF0A962503E69147CF

On the top of the page, below the total and address(es) headers, you'll find the **CSV EXPORT** button that leads to the CSV exporting page.

https://www.asastats.com/tax/2EVGZ4BGOSL3J64UYDE2BUGTNTBZZZLI54VUQQNZZLYCDODLY33UGXNSIU
https://www.asastats.com/tax/540A5D8CEC896E073F9170AF0A962503E69147CF/

Follow the instructions found on the page to get your CSV file(s( processed. Once the archive file is ready, the address/bundle page's CSV EXPORT button will change indicating that your archive file is ready for download.

Please bear in mind that this is an alpha version of the software. Only [Koinly](https://koinly.io/) tax software provider has been implemented so far and only one report preprocessing option is available (whether to use our maximum value engine or not). Please use our [GitHub issues](https://github.com/asastats/channel/issues) or our community channels to report bugs or to request features or check our [cumulative issue](https://github.com/asastats/channel/issues/1063).

**Comment by evertdiaz** [comment]
Hi @ipaleka, I'm the devRel assigned to review your proposal. I have one question: the milestone description mentions the next:

> CSV files will contain all the relevant transactions for any period user selects

I'm not able to find an option to select a specific period on the website, is this something still in progress? I'd appreciate a clarification here. 
Thanks!

**Comment by ipaleka** [comment]
> Hi @ipaleka, I'm the devRel assigned to review your proposal. I have one question: the milestone description mentions the next:
> 
> > CSV files will contain all the relevant transactions for any period user selects
> 
> I'm not able to find an option to select a specific period on the website, is this something still in progress? I'd appreciate a clarification here. Thanks!

At the time of writing the proposal, we didn't have full insight into the proper procedure of tax software providers and their usage of CSV files.

The point is that the tax software simply has to have all the data from the beginning in order to create a tax report for any period. If we limit our CSV files to a certain date range then the tax provider will mark the related funds as with an unknown source (or in a similar way). By providing all the data from the account creation date to the present day, the tax software can define any transaction to be taxable or not (for example, based on label user sets combined with the time when the funds arrived in the account).

TL; DR The only proper way to fully leverage the CSV files is to provide all the data and then the tax software can create a tax report for the period the user selects.

**Comment by evertdiaz** [comment]
Yeah it seems to be the best decision based on the new info.
Understood. Thanks for clarifying!
